joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

arm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - body_rotate_joint
    - elevation_joint
    - arm1_joint
    - arm2_joint
    - arm3_joint
    - arm4_joint
    - arm5_joint
    - arm6_joint

  constraints:
    goal_time: 6.0
    stopped_velocity_tolerance: 0.05
    body_rotate_joint: {trajectory: 0.1, goal: 0.1}
    arm1_joint: {trajectory: 0.1, goal: 0.1}
    arm2_joint: {trajectory: 0.1, goal: 0.1}
    arm3_joint: {trajectory: 0.1, goal: 0.1}
    arm4_joint: {trajectory: 0.1, goal: 0.1}
    arm5_joint: {trajectory: 0.1, goal: 0.1}
    arm5_joint: {trajectory: 0.1, goal: 0.1}
  stop_trajectory_duration: 0.5
  state_publish_rate:  125
  action_monitor_rate: 10

body_rotate_controller:
  type: position_controllers/JointPositionController
  joint: body_rotate_joint

elevation_controller:
  type: position_controllers/JointPositionController
  joint: elevation_joint

arm_base_controller:
  type: position_controllers/JointPositionController
  joint: arm_base_joint

gripper_controller:
  type: position_controllers/JointPositionController
  joint: gripper_joint

screen_tilt_controller:
  type: position_controllers/JointPositionController
  joint: head_screen_tilt_joint

pan_controller:
  type: position_controllers/JointPositionController
  joint: head_pan_joint

tilt_controller:
  type: position_controllers/JointPositionController
  joint: head_tilt_joint
